example_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The logr Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package glogr_test
+
+import (
+	"errors"
+	"flag"
+	"os"
+	"sync"
+
+	"github.com/go-logr/glogr"
+)
+
+var glogInit = sync.Once{}
+
+func initGlog() {
+	glogInit.Do(func() {
+		_ = flag.Set("v", "1")
+		_ = flag.Set("logtostderr", "true")
+		flag.Parse()
+	})
+	os.Stderr = os.Stdout
+}
+
+var errSome = errors.New("some error")
+
+func ExampleNew() {
+	initGlog()
+	log := glogr.New()
+	log.Info("info message with default options")
+	log.Error(errSome, "error message with default options")
+	log.Info("invalid key", 42, "answer")
+	log.Info("missing value", "answer")
+}
+
+func ExampleNew_withName() {
+	initGlog()
+	log := glogr.New()
+	log.WithName("hello").WithName("world").Info("thanks for the fish")
+}
+
+func ExampleNewWithOptions() {
+	initGlog()
+	log := glogr.NewWithOptions(glogr.Options{LogCaller: glogr.Error})
+	log.Info("Info log with LogCaller=Error")
+	log.Error(nil, "Error log with LogCaller=All")
+}
```

Comment:
Is there a way to see the output?

After a brief search I came up with none.

Reply:
Not that I can see - the timestamp is not disableable or fakeable.

Reply:
I could add it as a comment, but not a standard "Output" comment (ie. not test-verifiable)